tests/test_easy_config.py

```diff
@@ -47,6 +48,18 @@ def test_load_environment(example_env):
     assert ExampleConfig._load_environment() == {'number': 4, 'flag': True}
 
 
+def test_load_from_env(example_config_env):
+    """Test EasyConfig._load_environment."""
+    assert 'MYPROGRAM_FLAG' not in os.environ
+    assert 'MYPROGRAM_NUMBER' not in os.environ
```

Comment:
Can you assert with a message or add a comment here about why you're doing these assertions?

Reply:
yeah, but do you think it's better to remove them now that the issue i was checking for has been resolved?

Reply:
Honestly I'm happy to leave them as a reminder of what happened if it happens again.